examples/images_contours_and_fields/streamplot_demo_features.py

```diff
@@ -31,3 +31,56 @@
 ax2.streamplot(X, Y, U, V, density=0.6, color='k', linewidth=lw)
 
 plt.show()
+
+"""
+================================
+Streamplot function with masking
```

Comment:
You'll need to merge this docstring with the one at the beginning to make something coherent.